galasa-ui/src/tests/header.test.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import HomePage from '@/app/page';
+import PageHeader from '@/components/PageHeader';
+import { act, render, screen } from '@testing-library/react';
+
+jest.mock('next/headers', () => ({
+  cookies: jest.fn(() => ({
+    get: jest.fn().mockReturnValue('')
+  }))
+}))
+
+
+afterEach(() => {
+    delete process.env.NEXT_PUBLIC_GALASA_SERVICE_NAME
+})
```

Comment:
Won't need this if the service name is given as a property to the PageHeader component